src/Microsoft.Health.Fhir.Anonymizer.Stu3.Core/Microsoft.Health.Fhir.Anonymizer.Stu3.Core.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="MathNet.Numerics" Version="4.11.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.2" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Presidio" Version="0.0.1" />
```

Comment:
Is this the version we want of Presidio? I might be missing something...

Reply:
This is the unofficial nuget